tests/e2e/helpers/mock-console.js

```diff
@@ -1,10 +1,17 @@
 /**
  * Suppresses errors concerning web server already shut down.
- *
```

Comment:
Lots of empty line changes. Probably through the eslint-jsdoc-plugin update? If its a new default, maybe do that in a seperate PR to keep the git history cleaner?

Reply:
> Probably through the eslint-jsdoc-plugin update?

think so ...

will revert the second commit here and make second PR.